src/fides/data/sample_project/privacy_center/config/config.json

```diff
@@ -13,6 +13,16 @@
       "description_subtext": [],
       "identity_inputs": {
         "email": "required"
+      },
+      "custom_privacy_request_fields": {
+        "first_name": {
+          "label": "First name",
+          "required": true
+        },
+        "last_name": {
+          "label": "Last name",
+          "required": false
+        }
```

Comment:
In future if the env vars for this are false we should not show this in the privacy center